### ISSUE TITLE
FVS: ODBC error when generating FVSIn.db in same session that project was created

### ISSUE THREAD

**Issue by lbross** [USFS-PNW/Fia-Biosum-Manager#309]
From Dan Swann: Just to reiterate from our recent Teams call, when this error has popped up for me, I have had 100% success if I close BioSum, reopen my project, and click "Create FVS Input database file" again. It has successfully created the FVSIn database file every time I do that.

With debugging turned on, I do get a brief debug report back that reads:

//
//ExtractFIA2FVSRecords
//
Created DSN for FIABIOSUM_FIA2FVS_INPUT

![image](https://github.com/USFS-PNW/Fia-Biosum-Manager/assets/11527966/caf0da32-8921-4f57-b699-97a3ab33fa42)

**Comment by lbross** [comment]
I found some code related to registry and ODBC keys that served no purpose. Deleted it. I also added code to delete the ODBC DSN from the plot load process that could be pointed to the same FIA database. Because this only seems to happen when creating the FVSIn.db in the same session as a new project/loading plots, I suspect that something is hanging around from previous actions. The observation that FVSIn works after restarting BioSum supports this hypothesis. Unfortunately I was unable to recreate the problem so I'm not sure if these changes fixed it.
